src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.LegacyEditor/RazorWrapperFactory.TagHelperDocumentContextWrapper.cs

```diff
@@ -12,7 +12,7 @@ private class TagHelperDocumentContextWrapper(TagHelperDocumentContext obj) : Wr
     {
         private ImmutableArray<IRazorTagHelperDescriptor> _tagHelpers;
 
-        public string Prefix => Object.Prefix;
+        public string Prefix => Object.Prefix!;
```

Comment:
To avoid the possibility of Web Tools encountering a new nullability warning, I opted to use a `!` here rather than change the API.

Reply:
Yeah, you're right. I just did a survey through the Web Tools code and the only references to Prefix already handle null. I'll go ahead and update the API. It doesn't look to me like there's a compile-time break.